docs/docsite/rst/user_guide/playbooks_reuse.rst

```diff
@@ -1,72 +1,175 @@
 
-The bare ``include`` task (which was used for both Task files and Playbook-level includes) is still available, however it is now considered *deprecated*.
+   ========================= ======================================== ========================================
 
-Differences Between Dynamic and Static
-``````````````````````````````````````
+Re-using tasks as handlers
+==========================
 
-The two modes of operation are pretty simple:
+You can also use includes and imports in the :ref:`handlers` section of a playbook. For instance, if you want to define how to restart Apache, you only have to do that once for all of your playbooks. You might make a ``restarts.yml`` that looks like:
 
-* Dynamic includes are processed during runtime at the point in which that task is encountered.
-* Ansible pre-processes all static imports during Playbook parsing time.
+.. code-block:: yaml
 
-When it comes to Ansible task options like ``tags`` and conditional statements (``when:``):
+   # restarts.yml
+   - name: restart apache
+     service:
+       name: apache
+       state: restarted
 
-* For dynamic includes, the task options will *only* apply to the dynamic task as it is evaluated, and will not be copied to child tasks.
-* For static imports, the parent task options will be copied to all child tasks contained within the import.
+   - name: restart mysql
+     service:
+       name: mysql
+       state:restarted
 
-.. note::
-    Roles are a somewhat special case. Prior to Ansible 2.3, roles were always statically included via the special ``roles:`` option for a given play and were always executed first before any other play tasks (unless ``pre_tasks`` were used). Roles can still be used this way, however, Ansible 2.3 introduced the ``include_role`` option to allow roles to be executed inline with other tasks.
+You can trigger handlers from either an import or an include, but the procedure is different for each method of re-use. If you include the file, you must notify the include itself, which triggers all the tasks in ``restarts.yml``. If you import the file, you must notify the individual task(s) within ``restarts.yml``. You can mix direct tasks and handlers with included or imported tasks and handlers.
 
-Tradeoffs and Pitfalls Between Includes and Imports
-```````````````````````````````````````````````````
+Triggering included (dynamic) handlers
+--------------------------------------
 
-Using ``include*`` vs. ``import*`` has some advantages as well as some tradeoffs which users should consider when choosing to use each:
+Includes are executed at run-time, so the name of the include exists during play execution, but the included tasks do not exist until the include itself is triggered. To use the ``restart apache`` task with dynamic re-use, you must refer to the name of the include itself. This approach triggers all tasks in the included file as handlers. For example, with the task file shown above:
```

Comment:
remove "you must" e.g. "To use the ``restart apache`` task with dynamic re-use, refer to the name of the include itself. "